### YamlMime:PythonClass
uid: azure.data.tables.RequestTooLargeError
name: RequestTooLargeError
fullName: azure.data.tables.RequestTooLargeError
module: azure.data.tables
inheritances:
- azure.data.tables._error.TableTransactionError
summary: An error response with status code 413 - Request Entity Too Large
constructor:
  syntax: RequestTooLargeError(**kwargs)
methods:
- uid: azure.data.tables.RequestTooLargeError.add_note
  name: add_note
  summary: "Exception.add_note(note) \u2013\nadd a note to the exception"
  signature: add_note()
- uid: azure.data.tables.RequestTooLargeError.raise_with_traceback
  name: raise_with_traceback
  summary: 'Raise the exception with the existing traceback.


    Deprecated since version 1.22.0: This method is deprecated as we don''t support
    Python 2 anymore. Use raise/from instead.'
  signature: raise_with_traceback() -> None
- uid: azure.data.tables.RequestTooLargeError.with_traceback
  name: with_traceback
  summary: "Exception.with_traceback(tb) \u2013\nset self.__traceback__ to tb and\
    \ return self."
  signature: with_traceback()
attributes:
- uid: azure.data.tables.RequestTooLargeError.args
  name: args
- uid: azure.data.tables.RequestTooLargeError.continuation_token
  name: continuation_token
  signature: 'continuation_token: str | None'
- uid: azure.data.tables.RequestTooLargeError.error
  name: error
  signature: 'error: ODataV4Format | None'
- uid: azure.data.tables.RequestTooLargeError.exc_msg
  name: exc_msg
  signature: 'exc_msg: str'
- uid: azure.data.tables.RequestTooLargeError.exc_traceback
  name: exc_traceback
  signature: 'exc_traceback: TracebackType | None'
- uid: azure.data.tables.RequestTooLargeError.exc_type
  name: exc_type
  signature: 'exc_type: Type[Any] | None'
- uid: azure.data.tables.RequestTooLargeError.exc_value
  name: exc_value
  signature: 'exc_value: BaseException | None'
- uid: azure.data.tables.RequestTooLargeError.inner_exception
  name: inner_exception
  signature: 'inner_exception: BaseException | None'
- uid: azure.data.tables.RequestTooLargeError.message
  name: message
  signature: 'message: str'
- uid: azure.data.tables.RequestTooLargeError.model
  name: model
  signature: 'model: Any | None'
- uid: azure.data.tables.RequestTooLargeError.reason
  name: reason
  signature: 'reason: str | None'
- uid: azure.data.tables.RequestTooLargeError.response
  name: response
  signature: 'response: _HttpResponseCommonAPI | None'
- uid: azure.data.tables.RequestTooLargeError.status_code
  name: status_code
  signature: 'status_code: int | None'
